test/parallel/test-event-emitter-remove-listeners.js

```diff
@@ -83,3 +83,29 @@ e5.once('removeListener', common.mustCall(function(name, cb) {
 }));
 e5.removeListener('hello', listener1);
 assert.deepEqual([], e5.listeners('hello'));
+
+var e6 = new events.EventEmitter();
+count = 0;
+
+function listener3() {
```

Comment:
You don't need `count`. You can use `common.mustCall(fn, n)` where `fn` is the function and `n` is the number of times it must be called.